build.config.ts

```diff
@@ -1,6 +1,7 @@
+import type { BuildConfig } from 'unbuild'
 import { defineBuildConfig } from 'unbuild'
 
-export default defineBuildConfig({
+const build: BuildConfig[] = defineBuildConfig({
```

Comment:
I don't like that, feels a bit redundant. I want to keep my code close to UnJS repos... What is speaking against `export default defineBuildConfig()`?

Reply:
isolatedDeclarations these are mandatory income